gradle/libs.versions.toml

```diff
@@ -36,13 +36,11 @@ androidx-workManager = "2.8.1"
 androidx-browser = "1.5.0"
 
 # Compose
-compose = "1.6.0-alpha07"
-compose-material = "1.6.0-alpha07"
-compose-activity = "1.7.2"
+composeBom = "2023.10.00"
 compose-compiler = "1.5.2"
 compose-constraint = "1.0.1"
 compose-material3 = "1.2.0-alpha09"
-compose-navigation = "2.6.0"
+compose-navigation = "2.7.4"
```

Comment:
question: is it necessary? have you checked if it works well with "compose destinations" and we don't need to update it as well? 😄

Reply:
revered